test/tools/test_driver.sh

```diff
@@ -463,8 +463,7 @@ if [ ! -d \$CLM_TESTDIR ]; then
     fi
 fi
 
-## MCT and PIO build directorys
-export MCT_LIBDIR=\$CLM_TESTDIR/mct
+## PIO build directory
```

Comment:
@ekluzek commented:
Possibly remove this file altogether because it's deprecated and does not work.

Reply:
Today's ctsm software meeting: Yes, remove it.